spec.md

```diff
@@ -58,6 +58,36 @@ The email-body SHOULD be converted to plain text, full-quotes and similar region
 Attachments SHOULD be shown where possible.  If an attachment cannot be shown, a non-distracting warning SHOULD be printed.
 
 
+# Forwarded messages
+
+Forwarded messages are outgoing messages that contain a forwarded-header
+before the user generated content.
+
+The forwarded header MUST contain two lines: 
+The first line contains the text
+`---------- Forwarded message ----------`
+(10 minus, space, text `Forwarded message`, space, 10 minus). 
+The second line starts with `From: ` followed by the original sender
+which SHOULD be anonymized or just a placeholder.
+
+    From: sender@domain
+    To: rcpt@domain
+    Chat-Version: 1.0
+    Content-Type: text/plain
+    Subject: Chat: Forwarded message
+
+    ---------- Forwarded message ----------
+    From: Messenger
+
+    Hello world!
+
+Incoming forwarded messages are detected by the header.
+The messenger SHOULD mark these messages in a way that it becomes obvious
+that the message is not created by the sender.
+The original sender SHOULD NOT be displayed as it will be anonymized by most
+messengers anyway.
```

Comment:
i'd downgrade the last sentence to a note.  Maybe like this: 

Note that most messengers do not show the original sender with forwarded messages but MUAs typically expose the sender in the UI.

Reply:
k, did so :)